src/pages/Books/Books.module.scss

```diff
@@ -8,3 +8,22 @@
     gap: 3rem;
   }
 }
+
+.message {
+  text-align: center;
+  margin-top: 4rem;
+  padding: 2rem;
+  font-size: 1.25rem;
+}
+
+.welcome {
+  display: flex;
+  justify-content: center;
+  margin-top: 4rem;
+  font-size: 1.5rem;
+}
+
+.welcomeImage {
+  width: 300px;
```

Comment:
Our smallest supported screen size (a very good question whenever you start on a new project) is 320px. Make sure, that all padding and margins of the wrappers will STILL be able to fit this 300px welcome image into a smallest 320px screen